src/Stateless/TransitionBase.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Stateless
```

Comment:
Yes, that should be there... It doesn't show up in Solution Explorer, I'll remove this!